sbom/sbom.proto

```diff
@@ -36,6 +36,30 @@ enum Status {
   // are missing, preventing the BOM from being usable or reliable. A failed
   // status typically requires a thorough review and reprocessing of the BOM.
   STATUS_FAILED = 3;
+
+  // STARTED indicates that the SBOM scan was initialized. It is used for tracking
+  // progress of a scan over time.
+  STATUS_STARTED = 4;
+}
+
+// ProgressMessage 
+message ProgressMessage {
+  // Time of the progress message
+  string timestamp = 1;
+  // The external id associated with the scan error.
+  ExternalIDType externalID = 2;
+  // Status, e.g. "STARTED" being communicated
+  Status status = 3;
+}
+
+// ErrorReport is an error that occurs when performing an SBOM scan
+message ErrorReport {
+  // The error message from the scan.
+  string msg = 1;
+  // The external id associated with the scan error.
+  ExternalIDType externalID = 2;
+  // Status, e.g. "FAILED" being communicated
+  Status status = 3;
```

Comment:
Should we simply add the error msg field to the sbom message since the other fields are already available in Sbom message?

Reply:
👍

Reply:
updated!